src/components/BreadcrumbTrail.tsx

```diff
@@ -56,7 +56,7 @@ const BreadcrumbTrail = () : JSX.Element | null => {
 
     return (
         <div className={ classes.root }>
-            <Breadcrumbs aria-label="breadcrumb" maxItems={ isSmallScreen ? 2 : 6 } separator={ <NavigateNextIcon fontSize="small" /> }>
+            <Breadcrumbs aria-label="breadcrumb" maxItems={ isSmallScreen ? 2 : 8 } separator={ <NavigateNextIcon fontSize="small" /> }>
```

Comment:
Doesn't this mean that it will render 8 items on mobile, which I guess is too many to display on a single line?

Reply:
Oh, I see how it works now. But that doesn't really solve the problem. Because on a wide screen, you can fit more than 8 items, and on a medium screen, you could fit maybe 6 items, while on a mobile you would probably want to collapse to two.

What was the thought behind changing from 1200 to 1280?